frontend/package.json

```diff
@@ -93,5 +94,9 @@
     "> 1%",
     "last 2 versions",
     "not ie <= 8"
-  ]
+  ],
+  "devDependencies": {
+    "@babel/core": "^7.1.2",
```

Comment:
Don't use devDependencies - everything is dev (or not) since we build the final JS file :)